docs/src/templates/drawerActionBar.js

```diff
@@ -323,7 +325,10 @@ class ModulesDrawerActionBar extends React.Component {
                         </Header.Subheader>
                     </Header>
 
-                    Optional and Required Properties for <code>columns</code>:
+                    Optional and Required Properties for
+                    {' '}
+                    <code>columns</code>
+                    :
```

Comment:
This one (ESLINT insisting on all these things on separate lines) drives me a little bit nuts sometimes.  But c'est la vie.